sns/cleric-sns.json

```diff
@@ -1391,16 +1391,6 @@
         "Your insight and sympathy more than make up for your lack of formal medical training. You gain an expertise die on Medicine checks. In addition, you may always choose to use Wisdom when making Wisdom checks."
       ]
     },
-    {
-      "name": "Faithful Historian",
-      "source": "sns",
-      "featureType": [
-        "SF"
-      ],
-      "entries": [
-        "You have a deep relationship with religious history and art. You gain an expertise die on History checks. In addition, you may always choose to use Wisdom when making History checks."
-      ]
-    },
```

Comment:
Let me know if you want to not delete this. I think Cleric is actually one of the classes with good knack options so it's okay, and this is now identical to Theologian (bottom change in this file).

Reply:
Maybe add one for Medicine as Wis?